_vendor/github.com/chef/chef-server/docs-chef-io/content/server/ctl_chef_server.md

```diff
@@ -102,8 +102,8 @@ chef-server-ctl help
 
 ## install
 
-The `install` subcommand is used to install premium features of the Chef
-Infra Server: Chef management console and Chef Infra Client run
+The `install` subcommand is used to install premium feature of the Chef
```

Comment:
It looks like most of the features associated with this method are deprecated or not recommended. What do we use this endpoint for and can you help me get examples @tas50 ?
See the note on https://docs.chef.io/server/ctl_chef_server/#install

Do we still use it for Chef Infra Client run reporting? 

Otherwise:
```suggestion
The `install` subcommand is used to install Chef Infra Server replication and high availability configurations.
```

Reply:
We need to make this change in the server repo.